docs/source/guide/lre-5-theory.md

```diff
@@ -0,0 +1,85 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+
+```{warning}
+The user guide for LRE in Mitiq is currently under construction.
+```
+
+# What is the theory behind LRE?
+
+Layerwise Richardson Extrapolation (LRE), an error mitigation technique, introduced in
+{cite}`Russo_2024_LRE` works by creating multiple noise-scaled variations of the input
+circuit such that the noiseless expectation value is extrapolated from the execution of each
+noisy circuit.
+
+Similar to [ZNE](zne.md), this process works in two steps:
+
+- **Step 1:** Intentionally create multiple noise-scaled but logically equivalent circuits through unitary folding.
```

Comment:
I am going to skip using the word 'layerwise' as we have a similar submodule in ZNE. There a user can choose which layers they want to fold.

Reply:
Makes sense! I still think it might be nice to find a way to throw something in here to help someone understand &mdash; even if it's from a very high-level perspective &mdash; how these circuits are different from noise-scaled circuits in ZNE.